core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/diskModels.scala

```diff
@@ -91,3 +92,17 @@ object DiskType extends Enum[DiskType] {
     def googleString(googleProject: GoogleProject, zoneName: ZoneName): String = asString
   }
 }
+
+sealed trait FormattedBy extends EnumEntry with Product with Serializable {
+  def asString: String
+}
+object FormattedBy extends Enum[FormattedBy] {
+  val values = findValues
+
+  final case object Runtime extends FormattedBy {
```

Comment:
shall we call this Gce? just to be more specific

Reply:
yeah that makes sense, i can update